hdfs3/_pyarrow.py

```diff
@@ -0,0 +1,12 @@
+from __future__ import absolute_import
+
+import pyarrow as pa
+
+
+class HDFSWrapper(pa.filesystem.DaskFileSystem):
```

Comment:
Why is this required when isdir and isfile are already methods of DaskHDFileSystem? Can you derrive from both classes instead?

Reply:
Pyarrow requires classes to be subclasses of its filesystem. This is how other pyarrow classes wrapper classes are handled. Subclassing from both might also work but wouldn't necesssarily be simpler. This class should also disappear on a soon-ish basis.